operator/src/main/java/oracle/kubernetes/operator/helpers/DomainTopology.java

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) 2019, 2020, Oracle Corporation and/or its affiliates.
```

Comment:
New file ? - copyrights should be 2020 only.

Reply:
It's actually extracted from its original location in ConfigMapHelper, so the code is older than the file, which is why I did this, but I think you are correct. The dates should go with the file.